s3_data_management/s3_data_management.py

```diff
@@ -10,54 +10,57 @@
 
 from boto3 import client
 
-load_dotenv()
 TODAY = datetime.today()
 YESTERDAY = TODAY - timedelta(days=1)
 
 
-def get_bucket_keys(s3_client: client, folder_path: str,
-                    bucket_name: str = environ['BUCKET_NAME']) -> list:
-    """
-    Returns list of keys of csv files within a given s3 bucket, with a prefix matching the given
-    folder_path.
-    """
-    objects = s3_client.list_objects(Bucket=bucket_name, Prefix=folder_path).get('Contents')
+def create_s3_client():
+    """Creates a client that connects to s3 on AWS."""
+    return client("s3",
+                  aws_access_key_id=environ['AWS_ACCESS_KEY_ID'],
+                  aws_secret_access_key=environ['AWS_SECRET_ACCESS_KEY'])
+
+
+def get_bucket_keys(s3_client: client, folder_path: str, bucket_name: str) -> list:
+    """Returns a list of keys of csv files, with a prefix matching the folder_path."""
+    objects = s3_client.list_objects(
+        Bucket=bucket_name, Prefix=folder_path).get('Contents')
     if objects:
-        return [obj['Key'] for obj in objects if obj['Key'].split('.')[-1] == 'csv']
+        return [obj['Key'] for obj in objects if '.csv' in obj['Key']]
     return []
 
 
-def save_df_to_s3_bucket(s3_client: client, df: pd.DataFrame,
-                         key: str, bucket: str = environ['BUCKET_NAME']):
+def save_df_to_s3_bucket(s3_client: client, df: pd.DataFrame, key: str, bucket: str):
     """Function to save a pandas dataframe to the given s3 bucket."""
     file_body = df.to_csv()
-    s3_client.put_object(Body = file_body, Bucket = bucket, Key = key)
-
+    s3_client.put_object(Body=file_body, Bucket=bucket, Key=key)
 
-def combine_csv_files_for_month(s3_client: client, bucket_name: str = environ['BUCKET_NAME']):
-    """
-    Downloads relevant files from S3 to local, and then converts into two pandas dataframes, one
-    containing all recording info, and one all watering info.
-    """
 
+def combine_csv_files_for_month(s3_client: client, bucket_name: str):
+    """Downloads relevant files from S3 to local, and then converts into two pandas dataframes."""
```

Comment:
Should also mentioned that they're saved to file here; sorry, my fault, used to be two seperate functions and then I combined them because of redundant looping

Reply:
and that the old csv files are deleted

Reply:
yep sorry for not replying, have pushed the changed docstring!